src/libraries/GuardianUtils.sol

```diff
@@ -49,7 +49,7 @@ library GuardianUtils {
             revert IncorrectNumberOfWeights();
         }
 
-        if (threshold == 0 && guardianCount > 0) {
```

Comment:
You removed `&& guardianCount > 0` because `setupGuardians` will be never called in `configureRecovery` when no guardian is set, right?

Reply:
Yes exactly